graphql/schemas/Social/userList.graphql

```diff
@@ -0,0 +1,78 @@
+input ListInput {
+    name: String!
+    description: String
+    is_public: Boolean
+    is_default: Boolean
+}
+
+type List @model(class: "\\Kanvas\\Social\\UsersLists\\Models\\UserList"){
+    id: Int!
+    slug: String!
+    name: String!
+    description: String
+    is_public: Boolean
+    is_default: Boolean!
+    user: User!
+    apps: [App]
+    company: Company!
+    items: [Message]
+}
+
+extend type Mutation @guard {
+    createList(input: ListInput!): List 
```

Comment:
@FredPeal sorry for this update all the mutation to use user-list that way its easier to understand